src/main/java/bit/javaoop/AbstractManager.java

```diff
@@ -0,0 +1,53 @@
+package bit.javaoop;
+import java.util.List;
+
+public abstract class AbstractManager extends AbstractEmployee {
+
+    private final Float budget = 100_000f;
+    private List <AbstractEmployee> EmployeeList = getEmployeeList();
+
+
+    public AbstractManager(String name, Float salary)
+    {
+       super(name,salary);
+    }
+
+
+    public abstract boolean canHire();
+
+    public boolean isSatisfied()
+    {
+        return getSalary()>10_000.0f;
+    }
+
+
+    public void addEmployee(AbstractEmployee employee)
+    {
+        EmployeeList.add(employee);
+    }
+
+
+    public Float getSalaries() {
```

Comment:
Welcome to functional programming in Java where you can simply write:

```
return employeeList
    .stream()
    .mapToDouble(AbstractEmployee::getSalary)
    .sum();
```

Reply:
Also `getSalaries()` is a bad name. It does not describe what this method does. 
`getTotalSalary()` / `getSalarySum()` would be better :)